extension/docs/HowToRun.md

```diff
@@ -136,7 +136,8 @@ Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION
   "port": 8080,
   "keepAliveTimeout": 10000,
   "basicAuth" : true,
-  "removeSensitiveData": false
+  "removeSensitiveData": false,
+  "addCommercetoolsLineItems": true
 }
```

Comment:
I found it other configurations more useful. As all configs are categorised with titles: commercetools, adyen and others

Reply:
On pair review, we (@butenkor && @ahmetoz) decided to change all structure of describing environment variables, as it might be confusing as we can not reason all config, so we decided to separate it with optional and required environment variables.